include/mbgl/style/conversion/constant.hpp

```diff
@@ -113,6 +134,27 @@ struct Converter<std::array<float, 2>> {
 };
 
 template <>
+struct Converter<std::array<float, 3>> {
```

Comment:
I've been meaning to generalize this to a single templated `Converter<std::array<float, N>>` that can handle any size, and now's a good time. If you rebase on top of https://github.com/mapbox/mapbox-gl-native/commit/7c90db9241b47cb2b0a3ecd6a4d1df2fa1fd3f54 you can remove this addition.